summary_workflows/quantification/quantification_dockers/q2_validation/validation.py

```diff
@@ -2,17 +2,18 @@
 
 from __future__ import division, print_function
 import pandas
+import numpy as np
 import os, json
 import sys
 from argparse import ArgumentParser
 from JSON_templates import JSON_templates
 
 parser = ArgumentParser()
-parser.add_argument("-i", "--participant_data", help="list of cancer genes prediction", required=True)
+parser.add_argument("-i", "--participant_data", help="execution workflow prediction outputs", required=True)
 parser.add_argument("-com", "--community_name", help="name of benchmarking community", required=True)
-parser.add_argument("-c", "--challenge_types", nargs='+', help="list of types of cancer selected by the user, separated by spaces", required=True)
+parser.add_argument("-c", "--challenge_types", nargs='+', help="list challenges selected by the user, separated by spaces", required=True)
 parser.add_argument("-p", "--participant_name", help="name of the tool used for prediction", required=True)
-parser.add_argument("-r", "--public_ref_dir", help="directory with the list of cancer genes used to validate the predictions", required=True)
+parser.add_argument("-r", "--public_ref_dir", help="directory with the list of PAS ground truth used to validate the predictions", required=True)
```

Comment:
This directory is never used, see comment below. Furthermore, is it possible that the help message is not quite right? If anything, shouldn't it be about valid file formats and valid values within the files?